index.html

```diff
@@ -2290,7 +2326,7 @@ <h4 id="multibase">The `multibase` Datatype</h3>
             <dd>
 Any element of the lexical space is mapped to the value space by base-decoding
 the value based on the base-decoding alphabet associated with the
```

Comment:
Define the base-decoding alphabet and the meanings of its values.  The definitions needs to be specific enough to enable interoperable implementations.

That, or use of the base-decoding alphabet should be removed from the specification.

Reply:
That is done here:

https://pr-preview.s3.amazonaws.com/w3c/vc-data-integrity/pull/196.html#multibase-0

and here:

https://pr-preview.s3.amazonaws.com/w3c/vc-data-integrity/pull/196.html#base-encoding

We can add another algorithm section on generic base decoding, which when coupled with the Multibase section linked to above, provide enough information to provide interoperable implementations. Producing interoperable implementations around multibase has not been a challenge for implementers to date. That said, we have a lot of normative text in the specification that implementers can follow now.

If you disagree, please provide some concrete text suggestions.